suite/tests/tools.c

```diff
@@ -569,6 +569,35 @@ GLOBAL_LABEL(code_self_mod:)
 #endif
         END_FUNC(code_self_mod)
 
+#undef FUNCNAME
+#define FUNCNAME icache_sync
+        DECLARE_FUNC(FUNCNAME)
+GLOBAL_LABEL(FUNCNAME:)
+#ifdef X86
+        /* Nothing to do. */
+        ret
+#elif defined(ARM)
+        mov      r0, ARG1     /* start of region to flush */
+        add      r1, r0, #64  /* end of region to flush */
+        mov      r2, #0       /* flags: must be 0 */
+        push     {r7}
+        movw     r7, #0x0002  /* SYS_cacheflush bottom half */
+        movt     r7, #0x000f  /* SYS_cacheflush top half */
+        svc      #0           /* flush icache */
+        pop      {r7}
+        bx       lr
+#elif defined(AARCH64)
+        mov      x0, ARG1
+        dc       cvau, x0 /* Data Cache Clean by VA to PoU */
```

Comment:
Horizontal spacing to the left of the comment needs to be fixed so that the comment starts at the same column.

Reply:
Done.